blaze/server/server.py

```diff
@@ -70,6 +85,12 @@ class Server(object):
         A dictionary mapping dataset name to any data format that blaze
         understands.
 
+    formats : ``iterable[SerializationFormat]``, optional
+        An iterable of supported serialization formats. By default, the
+        server will support JSON.
+        A serialization format is an object that supports:
+        name, loads, and dumps.
```

Comment:
@llllllllll if we decide to go with the `dict` solution that @mrocklin showed above at some point in the future, would that break anything you're working on?